docs/locale/fr/LC_MESSAGES/userguide/parsec_workspaces.po

```diff
@@ -0,0 +1,214 @@
+msgid ""
+"If you have the **Owner** or **Manager** role, the ``Sharing and roles`` "
+"option will be available from the workspace menu."
+msgstr ""
+"Si vous avez le rôle **Propriétaire** ou **Gestionnaire**, l'option "
+"``Partage et rôles`` sera disponible dans le menu de l'espace de travail."
+
+#: ../../userguide/parsec_workspaces.rst:72
+msgid ""
+"Find the user you want to share the workspace with and select its :ref:"
+"`workspace roles <doc_userguide_parsec_workspaces_roles>`."
+msgstr ""
+"Trouvez l'utilisateur avec lequel vous voulez partager l'espace de travail "
+"et sélectionnez son :ref:`rôles d'espace de travail "
+"<doc_userguide_parsec_workspaces_roles>`."
+
+#: ../../userguide/parsec_workspaces.rst:-1
+msgid "Sharing a workspace"
+msgstr "Partager un espace de travail"
+
+#: ../../userguide/parsec_workspaces.rst:79
+msgid ""
+"Depending on the user profile within the organization, some roles may not be "
+"available for the selected user."
+msgstr ""
+"Selon le profil de l'utilisateur au sein de l'organisation, certains rôles "
+"peuvent ne pas être disponibles pour l'utilisateur sélectionné."
+
+#: ../../userguide/parsec_workspaces.rst:82
+msgid ""
+"You can remove access to this workspace by selecting ``Not shared`` for a "
+"given user."
+msgstr ""
+"Vous pouvez supprimer l'accès à cet espace de travail en sélectionnant ``Non "
+"partagé`` pour un utilisateur donné."
+
+#: ../../userguide/parsec_workspaces.rst:89
+msgid "Workspace roles"
+msgstr "Rôles des espaces de travail"
+
+#: ../../userguide/parsec_workspaces.rst:91
+msgid ""
+"The **workspace role** defines what the user is allowed to do within the "
+"workspace. Since the role is specific to the workspace, a user can have "
+"different roles in different workspaces."
+msgstr ""
+"Le **rôle de l'espace de travail** définit ce que l'utilisateur est autorisé "
+"à faire dans l'espace de travail. Comme le rôle est spécifique à l'espace de "
+"travail, un utilisateur peut avoir différents rôles dans différents espaces "
+"de travail."
+
+#: ../../userguide/parsec_workspaces.rst:95
+msgid "Here are the available roles:"
+msgstr "Voici les rôles disponibles :"
+
+#: ../../userguide/parsec_workspaces.rst:97
+msgid "**Reader**: Can view and open files."
+msgstr "**Lecteur** : Peut visualiser et ouvrir des fichiers."
+
+#: ../../userguide/parsec_workspaces.rst:98
+msgid "**Contributor**: Can edit, import and delete files."
+msgstr "**Contributeur** : Peut modifier, importer et supprimer des fichiers."
+
+#: ../../userguide/parsec_workspaces.rst:99
+msgid "**Manager**: Can manage user access to the workspace."
+msgstr ""
+"**Manager** : Peut gérer l'accès des utilisateurs à l'espace de travail."
+
+#: ../../userguide/parsec_workspaces.rst:100
+msgid ""
+"**Owner**: Can manage user access and re-encrypt the workspace. Can promote "
+"other users to Owner"
+msgstr ""
+"**Propriétaire** : Peut gérer l'accès des utilisateurs et re-chiffrer "
```

Comment:
Maybe mention that higher roles can also do what lower roles can do, ie a manager can also view/open/delete/import files.

Reply:
[`8af7270`](https://github.com/Scille/parsec-cloud/pull/8387/commits/8af72704163b00dc0eb1006ad9ed89e521f1731f)